src/modules/html.js

```diff
@@ -969,14 +969,24 @@ import { globalObject } from "../libs/globalObject.js";
    * @param {HTMLElement|string} source The source HTMLElement or a string containing HTML.
    * @param {Object} [options] Collection of settings
    * @param {function} [options.callback] The mandatory callback-function gets as first parameter the current jsPDF instance
+   * @param {number|array} [options.margin] Array of margins [left, bottom, right, top]
+   * @param {string} [options.filename] name of the file 
+   * @param {boolean} [options.enableLinks] should links work in the HTML when added to the PDF 
```

Comment:
The `enableLinks` option has no effect, we should remove it from the documentation